content/en/docs/kitex/Tutorials/advanced-feature/multi_service/multi_handler.md

```diff
@@ -0,0 +1,149 @@
+---
+title: "Generating Multiple Handlers For Multiple Services"
+date: 2024-09-12
+weight: 3
+keywords: ["Kitex", "Multiple Services", "Code Generation", "Multiple Handlers"]
+description: "Kitex supports generating handlers for each service in Multiple Services scenario and unifying registration."
+---
+
+> Support Version: Kitex Tool >= v0.11.0
+## Function Description
+
+Starting from v0.8.0, Kitex supports defining multiple Services in one idl (both thrift and protobuf are supported). For details, please refer to [Multiple Services](/docs/kitex/tutorials/advanced-feature/multi_service/multi_service). However, in the generated code, the handler.go corresponding to multiple Services will be overwritten, and only the Handler corresponding to the last Service in the idl will be retained.
```

Comment:
Why are you using an uppercase "S" for the word "service"? Unless it's a proper noun with a specific meaning, using a lowercase "s" might help avoid confusion.

Reply:
done.